app/views/episodes/_form_tags.html.erb

```diff
@@ -5,15 +5,10 @@
     </div>
 
     <div class="card-body">
-      <div class="prx-field-group">
-        <div class="form-floating">
-          <%= form.tag_select :categories, episode.categories, {}, data: {allow_new: true, allow_clear: true} %>
-          <%= form.label :categories %>
-        </div>
-
-        <button type="button" class="prx-btn-help" data-bs-toggle="popover" data-bs-content="<%= t(".help.categories") %>">
-          <span class="material-icons">help</span>
-        </button>
+      <div class="form-floating input-group">
+        <%= form.tag_select :categories, episode.categories, {}, data: {allow_new: true, allow_clear: true} %>
```

Comment:
This field still needs some work - bit buggy.  But... pre-existing condition.

Reply:
Write a ticket!

Reply:
I think #640 covers this?  Though I disagree with that "low" priority.